src/main/java/View/PlayBar.java

```diff
@@ -134,4 +131,26 @@ else if (actionEvent.getSource() == this.pause){
                 this.pause.setText("Pause");
         }
     }
+
+    /**
+     * Updates the play bar data.
+     */
+    public void update() {
+        InMemorySong song = (InMemorySong) viewModel.getPlaying();
+        this.song.setText(song.getName());
+        this.artist.setText(String.join(", ", song.getArtists()));
+        this.cover.setIcon(new ImageIcon(song.getCover().getScaledInstance(50, 50, Image.SCALE_DEFAULT)));
+        if (viewModel.getMediaType() == 2){
```

Comment:
future update might help to add a comment saying what numbers these refer to, i assume playlist vs space vs song (?), but ints by themselves aren't very descriptive